src/resources/config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Is this a testing artifact or just a sample file?

Reply:
just a sample file. presto provides sample configs which is very nice for getting started.